### PR TITLE
#170947421 setup sequelize ORM

### PR DESCRIPTION
**Pull request by hakoemmy**
#### What does this PR do?
- Have sequelize ORM configured
#### Description of Task to be completed?
 -  Install sequelize npm
- Setup migrations
- Setup default model
- Setup dummy user table
- Add documentation in readme
- Add some scripts commands in ```package.json```
#### How should this be manually tested?
- Just clone the [repo](https://github.com/andela/octopus-bn-backend.git) and 
- follow the instructions in [ReadMe](https://github.com/andela/octopus-bn-backend/blob/develop/README.md)
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#170947421](https://www.pivotaltracker.com/story/show/170947421)
### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: .sequelizerc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+require("@babel/register")
+const path = require('path')
+require('dotenv').config();
+
+module.exports = {
+  config: path.resolve('src/database/config', 'config.js'),
+  'models-path': path.resolve('src/database/models'),
+  'seeders-path': path.resolve('src/database/seeders'),
+  'migrations-path': path.resolve('src/database/migrations'),
+}
```
Comment (AbdoulNuru) [inline .sequelizerc]: @hakoemmy  ESLint

File: src/database/config/config.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  },
+
+  production: {
+    use_env_variable: 'DATABASE_URL',
+    dialect: 'postgresql',
+    operatorsAliases: false
+  }
+};
+
+module.exports = config;
```
Comment (blaise82) [inline src/database/config/config.js]: it's good thing that you used DATABASE_URL for connecting.
🔥

**Review by RusimbiPatrick — DISMISSED** [review]
Delete all unused variables like sequelize, models and their instances

File: .sequelizerc (hunk 1 above)
Comment (hakoemmy) [reply .sequelizerc]: @AbdoulNuru  thank you. Allow me to fix this

**Review by kcharles52 — APPROVED** [review]

**Review by blaise82 — APPROVED** [review]

**Review by RusimbiPatrick — APPROVED** [review]

**Review by izzett222 — APPROVED** [review]

**Review by AbdoulNuru — APPROVED** [review]

**Pull request merged**